### DISCUSSION TITLE
Batch / Bulk Conversion Function is Needed

### DISCUSSION THREAD

**Discussion by dcog989** [d2phap/ImageGlass#1858] [General]
https://github.com/d2phap/ImageGlass/issues/517

As per my comment on that (closed) issue - I was about to switch to ImageGlass but the lack of bulk / batch conversion is a killer. FastStone has it and so does every 'major' viewer I've looked at recently. It seems like a big omission in ImageGlass....

**Comment by hrxn** [comment]
It's still a viewer...
